app/services/bill-runs/two-part-tariff/submit-remove-bill-run-licence.service.js

```diff
@@ -0,0 +1,130 @@
+'use strict'
+
+/**
+ * Orchestrates removing a licence from a bill run whilst it is at the review stage
+ * @module SubmitRemoveBillRunLicenceService
+ */
+
+const { db } = require('../../../../db/db.js')
+
+/**
+ * Orchestrates removing a licence from a bill run whilst it is at the review stage. It does this by deleting all of the
+ * persisted data relating to the licence from the review tables. The licence will then be flagged for 2PT supplementary
+ * billing. If after removing a licence the bill run is empty, the bill run status will be set to set to `empty` and
+ * `true` returned so that the user is redirected back to the Bill runs page rather than Review licences.
+ *
+ * @param {string} billRunId - The UUID of the bill run that the licence is in
+ * @param {string} licenceId UUID of the licence to remove from the bill run
+ * @param {Object} sessionManager - The Hapi `request.yar` session manager passed on by the controller
+ */
+async function go (billRunId, licenceId, yar) {
+  await _removeChargeElementReturns(billRunId, licenceId)
+  await _removeReturns(billRunId, licenceId)
+  await _removeChargeElements(billRunId, licenceId)
+  await _removeChargeReferences(billRunId, licenceId)
+  await _removeChargeVersions(billRunId, licenceId)
+  await _removeLicence(billRunId, licenceId)
```

Comment:
I'd suggest moving these off into a separate function, for example, `_removeReviewData()`.